src/app/content/components/Page.tsx

```diff
@@ -92,17 +91,11 @@ export class PageComponent extends Component<PropTypes> {
   }
 
   public render() {
-    return <BookStyles>
-      {(className: string) => <MainContent className={[this.props.className, className].join(' ')}>
-        <div data-type='chapter'>
-          <div
-            data-type='page'
-            ref={(ref: any) => this.container = ref}
-            dangerouslySetInnerHTML={{ __html: this.getCleanContent()}}
-          />
-        </div>
-      </MainContent>}
-    </BookStyles>;
+    return <MainContent className={this.props.className}>
+      <div
```

Comment:
is this extra div super necessary?

Reply:
I only added it because `MainContent` did not allow `dangerouslySetInnerHTML` but I can try adding that prop to `MainContent`

Reply:
I tried moving it but `MainContent` would have 2 `ref=` which I wasn't sure how to handle